cba_settings_userconfig/cba_settings.sqf

```diff
@@ -152,13 +322,15 @@ force force ace_repair_engineerSetting_fullRepair = 2;
 force force ace_repair_engineerSetting_repair = 1;
 force force ace_repair_engineerSetting_wheel = 0;
 force force ace_repair_fullRepairLocation = 3;
-force force ace_repair_fullRepairRequiredItems = ["ToolKit"];
-force force ace_repair_miscRepairRequiredItems = ["ToolKit"];
+force force ace_repair_fullRepairRequiredItems = ["ace_repair_anyToolKit"];
+force force ace_repair_locationsBoostTraining = true;
+force force ace_repair_miscRepairRequiredItems = ["ace_repair_anyToolKit"];
```

Comment:
This is not the standard vanilla toolkit. Is there a reason why we go to a new toolkit? We have to change the engineers in our factions to have the right toolkit this way right?

Reply:
In previous versions of Arma, the "Toolkit" item was the only Toolkit, CDLC like SOG have added new Toolkit reskins so ACE have changed how the toolkit requirement is handled in order to be compatible with them. "Toolkit" is included within "ace_repair_anyToolKit"

Reply:
Ok clear! Thanks for clarifying this.